client/core/core.go

```diff
@@ -59,16 +56,17 @@ var (
 	// When waiting for a wallet to sync, a SyncStatus check will be performed
 	// ever syncTickerPeriod. var instead of const for testing purposes.
 	syncTickerPeriod = 10 * time.Second
+	// tickInterval is how often to check trades.
+	tickInterval = 60 * time.Second
```

Comment:
@buck54321 @itswisdomagain Reason for this is because tickInterval had an arbitrary formula based on the connected market's broadcast timeout.  Needing to have the `listen` goroutine, which also does the ticks, running from the start and thus before broadcast timeout is known made a problem.  The whole `tickCheckDivisions` thing was a bit questionable anyway.  I don't think I ever had a solid justification for a variable tick interval for markets with longer broadcast timeouts.

Reply:
It wasn't arbitrary. The broadcast timeout is an upper limit on what our tick interval should be. Basing the tick interval on the broadcast timeout is a reasonable approach. But using a 60s constant **is** arbitrary. What if the broadcast timeout is 30s and we never even get a second `tick`? Clearly, the operator that set the timeout is an ass, but that's not really the point.

Reply:
I mean the number of ticks per timeout was arbitrary, and the fact that tick rate was variable was awkward imo.
A broadcast timeout on the order of seconds is crazy as you've pointed out.
Alternatives to this?  I don't have a solution.

Reply:
Ya know, the `(*Ticker).Reset(time.Duration)` method that was added in 1.15 is now fair game for us.... let me see about that.  There are still likely some challenges though as the ticker's going to have to be in a `dexConnection` field to be able to reset it from the reconnect handler.

Reply:
Oh, nice. I didn't even know about `(*Ticker).Reset`.